roles-and-groups.md

```diff
@@ -0,0 +1,154 @@
+# Roles and Groups
+
+This file contains an overview of our rules for roles and groups.
+
+## Definition of general roles
+
+The Hiero project (and the LFDT) defines 3 different types of **roles**: **Contributors**, **Committers**, and **Maintainers**.
+Each person who contributes to Hiero must follow our [**Code of Conduct**](https://github.com/hiero-ledger/.github/blob/main/CODE_OF_CONDUCT.md).
+Those roles are not assigned globally for the Hiero project but per **project**.
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+
+Any person who contributes to a project is a Contributor. A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
+
+  - If a pull request has not received an approval within 2 days or a discussion has no activity for more than 2 days, the owner must reach out to the maintainers via our [hiero-general Discord channel](https://discord.lfdecentralizedtrust.org/).
```

Comment:
I think we should clarify business days

Reply:
I am assuming that when we measure if we respond to community PRs in  2 BD we are excluding PRs from maintainers/committers. I recently worked on a script that will get all issues from community excluding maintainers/individuals involved in the project I can share.

Reply:
Thank you!
I added that in my next patch..